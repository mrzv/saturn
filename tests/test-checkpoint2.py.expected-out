─────────────────────────── Skipping to checkpoint 2 ───────────────────────────
>>> a = 5                                                                       

─────────────────────────────────── Resuming ───────────────────────────────────
>>> print(a)                                                                    

5

