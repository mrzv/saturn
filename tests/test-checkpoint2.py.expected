a = 5

#chk>{{{
#chk>gANDIEfymvjN9EowcCGj+XNtin7uihFyNxXkbW2Myny7GZzJcQAugAN9cQAoWAwAAABfX2J1aWx0aW5z
#chk>X19xAWNidWlsdGlucwpfX2RpY3RfXwpYAQAAAGFxAksFdS4=
#chk>}}}

print(a)
#o> 5
